web/packages/teleport/src/Console/DocumentSsh/useFileTransfer.ts

```diff
@@ -54,7 +54,7 @@ export const useFileTransfer = (
   currentDoc: DocumentSsh,
   mfa: MfaState
 ) => {
-  const { filesStore } = useFileTransferContext();
+  const { filesStore, openedDialog } = useFileTransferContext();
```

Comment:
Optimally we wouldn't need to pass `openedDialog` like this, `DocumentSsh` would call `useFileTransferContext` to get it. That's just a nit for the future, as it might be that either `DocumentSsh`, `useFileTransferContext`, or `useFileTransfer` are written in a way that doesn't support that for some reason.